<ui version="4.0" >
 <class>ReferenceValuesForm</class>
 <widget class="QWidget" name="ReferenceValuesForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <property name="toolTip" >
   <string>Code_Saturne key word: P0</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBoxPressure" >
     <property name="title" >
      <string>Reference pressure</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <layout class="QHBoxLayout" >
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labelP0" >
          <property name="toolTip" >
           <string>Code_Saturne key word: P0</string>
          </property>
          <property name="text" >
           <string>Reference value for total pressure</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditP0" >
          <property name="toolTip" >
           <string>Code_Saturne key word: P0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelUnitP0" >
          <property name="text" >
           <string>Pa</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBoxTemperature" >
     <property name="title" >
      <string>Reference temperature</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <layout class="QHBoxLayout" >
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labelT0" >
          <property name="toolTip" >
           <string>Code_Saturne key word: T0</string>
          </property>
          <property name="text" >
           <string>Reference temperature</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditT0" >
          <property name="toolTip" >
           <string>Code_Saturne key word: T0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelUnitT0" >
          <property name="text" >
           <string>K</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" >
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labelInfoT0" >
          <property name="text" >
           <string>(used for initialization of enthalpy and density)</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QGroupBox" name="groupBoxMassMolar" >
     <property name="title" >
      <string>Mass molar molecular reference</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <layout class="QHBoxLayout" >
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labelMassMolar" >
          <property name="toolTip" >
           <string>Code_Saturne key word: WMOLME</string>
          </property>
          <property name="text" >
           <string>Mass molar reference</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditMassMolar" >
          <property name="toolTip" >
           <string>Code_Saturne key word: WMOLME</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelUnitMassMolar" >
          <property name="text" >
           <string>kg/mol</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" >
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labelInfoMassMolar" >
          <property name="text" >
           <string>(used for initialization of user scalars)</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
